<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="edit_list_of_devices">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>coma separated list of devices</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_commands">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>511</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>coma separated list of commands to run WITHOUT quotes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_export">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Export to .txt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 41, 41);
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login_execute">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Login an Execute</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_username">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_password">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>password</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ptedit_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Commands output</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>626</x>
      <y>670</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 243, 180);</string>
    </property>
    <property name="text">
     <string>https://shahzadqadir.net</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>670</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DevNet scripts, tutorial, videos</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>edit_list_of_devices</tabstop>
  <tabstop>edit_commands</tabstop>
  <tabstop>edit_username</tabstop>
  <tabstop>edit_password</tabstop>
  <tabstop>btn_login_execute</tabstop>
  <tabstop>btn_export</tabstop>
  <tabstop>btn_exit</tabstop>
  <tabstop>ptedit_output</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
